pandas/core/generic.py

```diff
@@ -1892,6 +1897,29 @@ def to_sql(self, name, con, schema=None, if_exists='fail', index=True,
             Optional specifying the datatype for columns. The SQL type should
             be a SQLAlchemy type, or a string for sqlite3 fallback connection.
 
+        Returns
```

Comment:
The Returns section can be ommitted if ther's no return value.

Reply:
python scripts/validate_docstrings.py pandas.DataFrame.to_sql complains:
Errors found:
	No returns section found